src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -97,9 +98,9 @@ CEvaluationResult* CCrossValidation::evaluate_impl()
 		SG_DEBUG("Creating CrossValidationStorage.\n")
 		CrossValidationStorage* storage = new CrossValidationStorage();
 		SG_REF(storage)
-		storage->set_num_runs(m_num_runs);
-		storage->set_num_folds(m_splitting_strategy->get_num_subsets());
-		storage->set_expose_labels(m_labels);
+		storage->put("num_runs", utils::safe_convert<index_t>(m_num_runs));
```

Comment:
minor: inside C++ code, I am not the biggest fan of using put with strings if the actual typed C++ API is available (setters). But not need to change that now